R/factset.protobuf.stachextensions/R/V2ColumnOrganizedStachUtilities.R

```diff
@@ -0,0 +1,80 @@
+#' @docType class
+#' @title V2ColumnOrganizedStachUtilities
+#' @description Provide helper functions for column organized stach
+#'
+#' @importFrom R6 R6Class
+#' @export
+V2ColumnOrganizedStachUtilities <- R6::R6Class(
+  "V2ColumnOrganizedStachUtilities",
+  public = list(
+
+    #' @description  Get Primary Table Ids
+    #' @param package  Stach Data which is represented as a Package object
+    #' @return Returns list of primary table ids
+    GetPrimaryTableIds = function(package) {
+      primaryTableIds = package[["primaryTableIds"]]
+
+      return(primaryTableIds)
+    },
+    #' @description  Get decompress stach data
+    #' @param package  Stach Data which is represented as a Package object
+    #' @return Returns decompressed stach data
+    DecompressAll = function(package) {
```

Comment:
Is it better to call this method just `Decompress` and `Decompress` to `DecompressTable`?

Reply:
Sure that is fine